lms/djangoapps/edraak_university/migrations/__init__.py

```diff
@@ -0,0 +1 @@
+# TODO: Link the UniversityID to the CourseEnrollment object to limit it for enrolled users only.
```

Comment:
It's a big oops, [there's a task for it](https://app.asana.com/0/172236337713853/247229010586490).